pkg/reconciler/knative/resources/knative_configuration.go

```diff
@@ -71,6 +71,33 @@ func (c *ConfigurationBuilder) CreateKnativeConfiguration(name string, metadata
 		annotations[autoscaling.ClassAnnotationKey] = autoscaling.KPA
 	}
 
+	// Pass the PVC and mountPath for Webhook
+	// TBD @jinchihe Need a handler to reduce code duplication.
+	if modelSpec.Tensorflow != nil {
+		if modelSpec.Tensorflow.PersistentVolumeClaim.Name != "" {
+			annotations["persistentVolumeClaim.name"] = fmt.Sprint(modelSpec.Tensorflow.PersistentVolumeClaim.Name)
```

Comment:
why not `annotations["persistentVolumeClaim.name"] = modelSpec.Tensorflow.PersistentVolumeClaim.Name` directly?

Reply:
The part has been removed in new solution, thanks.